sdk-extensions/tracing-incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/trace/zpages/TraceConfigzZPageHandler.java

```diff
@@ -254,8 +254,8 @@ private void emitActiveTable(PrintStream out) {
    */
   private void emitHtmlBody(PrintStream out) {
     out.print(
-        "<a href=\"/\"><img style=\"height: 90px;\" src=\"data:image/png;base64,"
-            + ZPageLogo.getLogoBase64()
+        "<a href=\"/\"><img style=\"height: 90px;\" src=\""
+            + ZPageLogo.getLogoPath()
```

Comment:
The base64 encoding of the logo and favicon didn't work because the tracing incubator module has animal sniffer enabled, and `java.util.Base64` isn't available in android 21. I've adjusted the code to add simple static resource handlers for `/logo.png` and `/favicon.png` instead.